ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -5,21 +5,105 @@
 
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { loading } from 'ember-loading';
 
 export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
   // =services
 
   @service store;
+  @service confirm;
+  @service ipc;
+  @service router;
+  @service session;
+
+  queryParams = {
+    isConnecting: {
```

Comment:
None blocking question: Within the controller above, `isConnecting` is defined as Boolean, but here we are setting it as object, can you elaborate?

Reply:
Here we are telling this route to refresh the model if the [query param changes](https://hashicorp.atlassian.net/browse/ICU-10519). It does not set the type of `isConnecting`. With that said, I am not sure we will even need this. I will test this route transition with and without setting `refreshModel` to true.

Reply:
ahh gotcha! my bad 😉

Reply:
No worries. I had to do a good bit of readying to make sure I understood this.